pkgs/misc/blesh/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchgit }:
+stdenv.mkDerivation rec {
+  pname = "blesh";
+  version = "0.3.3"; 
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "398e404d624ef6365d9175882e24f6e180d8b6b2";
```

Comment:
For version 0.3.3 this could be `"v${version}"`, but the listed revision is not actually a tagged version and so the `version` number should probably reflect that (i.e. `version = "0.4.0_devel3.398e404";` or something.)